src/Request/CliRequest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Olcs\Logging\Request;
+
+use Laminas\Stdlib\RequestInterface;
+use Olcs\Logging\CliLoggableInterface;
+
+class CliRequest implements CliLoggableInterface, RequestInterface
+{
+    public function setMetadata($spec, $value = null)
+    {
+    }
+
+    public function getMetadata($key = null)
+    {
+    }
+
+    public function setContent($content)
+    {
+    }
+
+    public function getContent()
+    {
+    }
+}
```

Comment:
This belongs in the `olcs-backend` repository. How come we don't extend: `\Laminas\Stdlib\Message` instead too?

Reply:
Removed.